resources/views/settings/profile.blade.php

```diff
@@ -28,13 +28,29 @@
                         <input type="text" name="settings[marketplace_name]" maxlength="32" placeholder="{{ __('profiles.fields.profile-name') }}" class="rounded border p-2 w-full" value="{!! old('settings[marketplace_name]', $user->settings['marketplace_name'] ?? null) !!}" />
                     </x-forms.field>
 
+                    @php $helper =  __('profiles.helpers.pronouns', [
+                        'marketplace' => '<a href="' . config('marketplace.url') . '">' . __('footer.plugins'). '</a>',
+                        'profile' => '<a href="' . route('users.profile', $user) . '" target="_blank">' . __('profiles.settings.helpers.profile') . '</a>'])
+                    @endphp
+                    <x-forms.field field="pronouns" :label="__('profiles.fields.pronouns')" :helper="$helper">
+                        <input type="text" name="settings[pronouns]" maxlength="45" placeholder="{{ __('profiles.fields.pronouns') }}" class="rounded border p-2 w-full" value="{!! old('settings[pronouns]', $user->settings['pronouns'] ?? null) !!}" />
+                    </x-forms.field>
+
                     @php $helper = __('profiles.settings.helpers.bio', [
     'link' => '<a href="' . route('users.profile', $user) . '" target="_blank">' . __('profiles.settings.helpers.profile') . '</a>'
     ]); @endphp
                     <x-forms.field field="bio" :label="__('profiles.fields.bio')" :helper="$helper">
                         <textarea name="profile[bio]" placeholder="{{ __('profiles.placeholders.bio') }}" class="w-full rounded border p-2" rows="5" maxlength="300">{!! old('profile[bio]', \Illuminate\Support\Arr::get($user->profile, 'bio')) !!}</textarea>
                     </x-forms.field>
 
+                    @php $helper =  __('profiles.helpers.link', [
+                        'marketplace' => '<a href="' . config('marketplace.url') . '">' . __('footer.plugins'). '</a>',
+                        'profile' => '<a href="' . route('users.profile', $user) . '" target="_blank">' . __('profiles.settings.helpers.profile') . '</a>'])
```

Comment:
remove _target blank, let the user decide if they want to open a new tab or not,